.pre-commit-config.yaml

```diff
@@ -0,0 +1,44 @@
+repos:
```

Comment:
Could you add the `ci` config from here please?

https://github.com/django-cms/django-cms/blob/develop/.pre-commit-config.yaml

And you can just remove the black section rather than comment out as we've no plans to use black

Reply:
I already removed the commented in my last commit.

Reply:
will add the ci now

Reply:
latest commit: 297069dec1f7ad2381c24355ba0f3807f88cedee